scipy/ndimage/measurements.py

```diff
@@ -571,6 +571,18 @@ def _sum_centered(labels):
         return (counts, sums, sums_c)
 
 
+def sum_labels(input, labels=None, index=None):
+    """
+    Calculate the sum of the values of the array.
+
+    Notes
+    -----
+    This is an alias for `ndimage.sum`.  See the ``sum``
+    docstring for more details.
+
```

Comment:
You can also add here "this alias is kept for backwards compatibility reasons, for new code please prefer `sum_labels."

Reply:
Done